pkg/yurtctl/cmd/convert/edgenode.go

```diff
@@ -242,7 +240,8 @@ func (c *ConvertEdgeNodeOptions) RunConvertEdgeNode() (err error) {
 			klog.Errorf("fail to run ServantJobs: %s", err)
 			return err
 		}
-	} else {
+	} else if len(c.EdgeNodes) == 0 || (len(c.EdgeNodes) == 1 && c.EdgeNodes[0] == nodeName) {
```

Comment:
how about add `nodeName != ""` for `len(c.EdgeNodes) == 0` condition?

Reply:
ok